templates/includes/install-problems.hbs

```diff
@@ -2,28 +2,42 @@
 <div id="InstallTroubleshooting" class="page-header">
   <h1>Installation Troubleshooting</h1>
 </div>
-
+<h2>Boot problems</h2>
 <div class="callout">
-    <h4>Commands fail or stop during the ~5 minute flashing process</h4>
-    Smartwatches are prone to lose a proper connection to their cradle over time due to sweat corroding the watch pins. This is a particularly a regular problem with dory (LG G Watch). Cleaning the pins with isopropyl alcohol or even carefully sanding them (e.g. with a nailfile) can restore a reliable connection.
+    <h3>Bootloop of any kind or stuck boot process</h3>
+    Don't panic. Non of the commands we advised you to use can damage or brick your watch permanently.<br>
+    You can always go back to the fastboot bootloader menu, by using the <a href="https://asteroidos.org/wiki/useful-commands/#boot-to-fastboot-bootloader-menu">manual finger combos for your watch</a>.
+    <ul>
+      <li><h4>Verify that you used the correct image files</h4>Some devices like the Asus Zenwatch 2, the Fossil Gen4 models and the MTK6580 watches have more than on system image supplied, for different variants with slightly different hardware. Most watches have a graphical fastboot menu that shows the correct codename for the watch. Others need to be identified using the hardware differences noted on the install page.</li>
+      <li><h4>Clean your watch and charger contacts</h4>Even in case the fastboot flash process succeeds with no errors, it can still silently fail due to connection problems caused by dirty contacts. Clean the pins and pads with isopropyl alcohol. Ideally using that old toothbrush you keep around for electronics cleaning or a microfiber cloth. Carefully sanding the contacts (e.g. with a nailfile) can help in case of obvious corrosion.</li>
+      <li><h4>Ensure a stable USB connection</h4>Put the watch and your computer on the same flat surface. Try to not move the cable connecting your computer and the watch during the whole flashing process. In case of worn or corroded contacts and pogo-pins that lost their springiness, it might be useful to attach the watch to its cradle or charging cable using ductape or rubber bands.</li>
+      <li><h4>Repeat the flash process.</h4>Up to five tries have been reported to possibly give a successful result in the end.</li>
```

Comment:
`Up to five tries have been reported to possibly give a successful result in the end.`
The phrasing here is a bit strange - I'd try something like:
`it can take up to five flashing attempts for a successful result`
or
`Some users have reported needing up to five flashing attempts for a successful result`
in whatever combination sounds best to you